ui/src/style/pdf-viewer.less

```diff
@@ -15,4 +15,14 @@
   */
 .linkAnnotation a[href^="#cite"] {
   display: none;
+}
+
+/** 
+  * Consistent highlight between default pdf.js highlight for tables, figures, etc. 
+  * and custom purple highlight for citation annotations.
+  */
+.linkAnnotation a:not([href^="#cite"]):hover {
```

Comment:
```suggestion
.linkAnnotation a:hover {
```

What do you think of making this selector apply to all PDF.js links? The advantage of this approach is that, if in the future we decide to disable citation annotations, then our styling will apply to PDF.js's links without us needing to make additional modifications.

Reply:
Apparently if we only use a:hover as our selector, this is less specific than the original pdf.js styling and it gets overwritten by the default yellow highlight. If we want to keep the purple highlight even if we disable our citation annotations, it would need to have the selector look something like this:
![1](https://user-images.githubusercontent.com/52334652/76265135-0117c800-6221-11ea-84bb-2b5c4ed8026c.png)

just wondering if this is something we want to have? will we ever decide to disable citation annotations in the future?

Reply:
> just wondering if this is something we want to have?

sure! I think the verbose selector you show above is okay.

> will we ever decide to disable citation annotations in the future?

I might want to turn it on and off for upcoming studies, though there are no concrete plans right now.